Minor fixes to Sentry Docs (#5357)
getsentry/sentry-docs: src/platforms/javascript/guides/capacitor/ionic.mdx

description: "Learn how to use Sentry with Ionic."
---

> Sentry Capacitor supports [Ionic](https://ionicframework.com/) out of the box, the documentation below is similar to the one at [Getting Started documentation](/platforms/javascript/guides/capacitor/) since the setup of both Capacitor and Ionic are the same without the need of any additional change.

Reviewer: Fixed some punctuation and wording. 

```suggestion
Sentry Capacitor supports [Ionic](https://ionicframework.com/) out of the box. The documentation below is similar to our [Capacitor Getting Started documentation](/platforms/javascript/guides/capacitor/) since the setup of Capacitor and Ionic are the same.
```
